### PR TITLE
chore: resources in separate yaml

### PR DESCRIPTION
**Pull request by ThibaultFy**
## Description

part of FL-1562

## How has this been tested?

Resources are tested by being injected in org-1 with skaffold

## Checklist

- [ ] [changelog](../CHANGELOG.md) was updated with notable changes
- [ ] documentation was updated

### CODE REVIEW

File: backend/substrapp/kubernetes_utils.py [hunk 1]
```diff
@@ -55,9 +55,17 @@ def get_resources_requirements_from_yaml(
     """Return a kubernetes.client.V1ResourceRequirements object from a yaml string."""
     resources_dict = yaml.safe_load(yaml_resources)
 
-    return kubernetes.client.V1ResourceRequirements(
-        requests=resources_dict["requests"], limits=resources_dict["limits"]
-    )
+    try:
+        requests = resources_dict["requests"]
+    except KeyError:
+        requests = None
```
Comment (guilhem-barthes) [inline backend/substrapp/kubernetes_utils.py:61]:
```suggestion
    requests = resources_dict.get("requests", None)
```

File: backend/substrapp/kubernetes_utils.py [hunk 2]
```diff
@@ -55,9 +55,17 @@ def get_resources_requirements_from_yaml(
...
+
+    try:
+        limits = resources_dict["limits"]
+    except KeyError:
+        limits = None
```
Comment (guilhem-barthes) [inline backend/substrapp/kubernetes_utils.py:66]:
```suggestion
    limits = resources_dict.get("limits", None)
```

File: skaffold.yaml [hunk 3]
```diff
@@ -28,6 +28,7 @@ deploy:
         chartPath: charts/substra-backend
         valuesFiles:
           - examples/values/backend-org-1.yaml
+          - charts/substra-backend/resources.yaml
```
Comment (thbcmlowk) [inline skaffold.yaml:31]: I am not sure I am following. Is it not prod values that are in the `resources.yaml` file?

Comment (ThibaultFy) [reply skaffold.yaml:31]: The values are the ones that we currently apply by default. It is up to us to decide if we put prod values (so with highest requests) or if we want to keep these low requests. Also, il was to test that it was working as expected. We can also completely remove it from skaffold but I don't know how to ensure that this file stay up to date with CI

**Comment by ThibaultFy** [comment]
close in favor of https://github.com/Substra/substra-backend/pull/907

**Pull request closed without merging**